dvc/fs/http.py

```diff
@@ -159,6 +160,17 @@ def chunks(fobj):
         if response.status_code not in (200, 201):
             raise HTTPError(response.status_code, response.reason)
 
+    def _upload_fobj_not_chunked(self, fobj, to_info):
+        response = self.request(self.method, to_info.url, files={"file": fobj})
+        if response.status_code not in (200, 201):
```

Comment:
Duplicated code with what in `_upload_fobj_chunked` maybe we can move them to `_upload_fobj`